integrationTests/vm/arwen/utils.go

```diff
@@ -134,6 +135,7 @@ func SetupTestContextWithGasSchedulePath(t *testing.T, gasScheduleConfigPath str
 	context.T = t
 	context.Round = 500
 	context.EpochNotifier = &epochNotifier.EpochNotifierStub{}
+	context.EnableEpochsHandler = &testscommon.EnableEpochsHandlerStub{}
```

Comment:
also here?

Reply:
still needed for the remaining components that still use it